download-artifact/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2020 Dawid Dziurla
```

Comment:
@dawidd6 Could this be updated/relicensed to Homebrew (and perhaps be the same license as Homebrew/brew?)

Reply:
Sure, just take it :smile: